lvdgmic plugin pack does not test if you have gmic installed

No check for gmic in plugin-packs/lvdgmic

:~/Downloads/veejay-git/veejay-current/plugin-packs/lvdgmic$ make
make  all-recursive
make[1]: Entering directory
`/home/zenny/Downloads/veejay-git/veejay-current/plugin-packs/lvdgmic'
Making all in liblvdgmic
make[2]: Entering directory
`/home/zenny/Downloads/veejay-git/veejay-current/plugin-packs/lvdgmic/liblvdgmic'
/bin/bash ../libtool  --tag=CXX   --mode=compile g++ -DHAVE_CONFIG_H
-I. -I.. -m64 -fPIC -DPIC -ftree-vectorize -O3 -ffast-math -Wall
-Wunused  -DSILENT    -g -O2  -MT lvdgmic.lo -MD -MP -MF
.deps/lvdgmic.Tpo -c -o lvdgmic.lo lvdgmic.cc
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I.. -m64 -fPIC -DPIC
-ftree-vectorize -O3 -ffast-math -Wall -Wunused -DSILENT -g -O2 -MT
lvdgmic.lo -MD -MP -MF .deps/lvdgmic.Tpo -c lvdgmic.cc  -fPIC -DPIC -o
.libs/lvdgmic.o
lvdgmic.cc:4:18: fatal error: gmic.h: No such file or directory
 #include <gmic.h>
                  ^
compilation terminated.
make[2]: *** [lvdgmic.lo] Error 1
make[2]: Leaving directory
`/home/zenny/Downloads/veejay-git/veejay-current/plugin-packs/lvdgmic/liblvdgmic'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory
`/home/zenny/Downloads/veejay-git/veejay-current/plugin-packs/lvdgmic'
make: *** [all] Error 2

--- a/veejay-current/plugin-packs/lvdgmic/configure.ac
+++ b/veejay-current/plugin-packs/lvdgmic/configure.ac
@@ -90,6 +90,15 @@ AM_PROG_AS
 
 AC_CHECK_HEADERS([fenv.h stdint.h inttypes.h sys/types.h alloca.h])
 
+AC_LANG_PUSH([C++])
+AC_CHECK_HEADER([gmic.h], have_gmic=true,have_gmic=false)
+AC_LANG_POP([C++])
+
+if test x$have_gmic = xfalse;
+then
+	AC_MSG_ERROR([gmic.h not found - please install gmic])
+fi
+
 dnl AX_PREFIXED_DEFINE([VEEJAY], [HAVE_STDINT_H])
 dnl AX_PREFIXED_DEFINE([VEEJAY], [HAVE_INTTYPES_H])
 

